### DISCUSSION TITLE
Create interactive Tracebacks with collapsed strack frames/variables ?!?

### DISCUSSION THREAD

**Discussion by jedie** [Textualize/textual#1239] [Ideas]
Is it's possible to build interactive TUI Tracebacks like Django or werkzeug?

e.g.:
![grafik](https://user-images.githubusercontent.com/71315/203010458-586544a5-276e-4688-9039-b6f6f318dc82.png)
from https://werkzeug.palletsprojects.com/en/2.2.x/debug/#using-the-debugger

Would be great if it's possible to use these TUI Tracebacks for unittests, isn't it?

**Comment by davep** [comment]
There isn't currently a method of doing this as part of Textual, but a more feature-rich [devtools](https://textual.textualize.io/guide/devtools/) is [on the roadmap](https://textual.textualize.io/roadmap/).

**Reply by willmcgugan** [reply]
It is a good idea though. Would make a cool project…

**Comment by jedie** [comment]
Has anybody tried this?
